app/views/users/show.html.haml

```diff
@@ -22,19 +22,17 @@
 
   = render_view_hook "additional_user_fields", f: f
 
-- if can? :edit, @user
-  %ul.inline
+%ul.inline
+  - if current_ability.can?(:switch_to, @user) && @user.active? && !current_facility.cross_facility?
+    %li= link_to text("search.results_table.order_for"), facility_user_switch_to_path(current_facility, @user), class: "btn btn-primary"
```

Comment:
I kind of feel like `btn-primary` is not the right styling. I'm not quite sure what is.

https://bootstrapdocs.com/v2.3.2/docs/base-css.html#buttons

Reply:
Also, can we put this after the "Edit" button?

Reply:
Maybe the edit button should just be a plain button? For the "edit" button for products we don't use `btn-primary`. What do you think?

Reply:
Yup, that makes sense to me - edit button will just be `btn`, and order for will be `btn-primary`, and come after the edit button.